Core/HLE/proAdhoc.cpp

```diff
@@ -983,6 +988,54 @@ void freeFriendsRecursive(SceNetAdhocctlPeerInfo * node) {
 	free(node);
 }
 
+//@params chatmenu pass NULL on destroy, and pass ChatMenu On Create (EmuScreen.cpp)
+void setChatPointer(ChatMenu * chatmenu) {
+	if (chatmenu != NULL) {
```

Comment:
`delete` does nothing if its argument is null.  So you don't need this check.

-[Unknown]